README.md

```diff
@@ -38,116 +38,53 @@ Here are a few key pages you might be interested in:
 
 ## Quickstart
 
-TigerBeetle is easy to run with or without Docker, depending on your
-preference. First, we'll cover running the [Single
-Binary](#single-binary). And below that is how to run [with
-Docker](#with-docker).
+First, grab a prebuilt copy of TigerBeetle.
```

Comment:
```suggestion
First, download a prebuilt copy of TigerBeetle.
```

"grab" is really nice/better for the feel, but we need to hint that this step will download a prebuilt binary rather than build locally.

Reply:
✅ pulled in locally